src/Aspire.Hosting.Azure.CosmosDB/api/Aspire.Hosting.Azure.CosmosDB.cs

```diff
@@ -8,25 +8,45 @@
 //------------------------------------------------------------------------------
 namespace Aspire.Hosting
 {
-    public partial class AzureCosmosDBResource : Azure.AzureProvisioningResource, ApplicationModel.IResourceWithConnectionString, ApplicationModel.IResource, ApplicationModel.IManifestExpressionProvider, ApplicationModel.IValueProvider, ApplicationModel.IValueWithReferences, ApplicationModel.IResourceWithEndpoints
+    public partial class AzureCosmosDBResource : Azure.AzureProvisioningResource, ApplicationModel.IResourceWithConnectionString, ApplicationModel.IResource, ApplicationModel.IManifestExpressionProvider, ApplicationModel.IValueProvider, ApplicationModel.IValueWithReferences, ApplicationModel.IResourceWithEndpoints, Azure.IResourceWithAzureFunctionsConfig
     {
         public AzureCosmosDBResource(string name, System.Action<Azure.AzureResourceInfrastructure> configureInfrastructure) : base(default!, default!) { }
 
         public Azure.BicepSecretOutputReference ConnectionString { get { throw null; } }
 
         public ApplicationModel.ReferenceExpression ConnectionStringExpression { get { throw null; } }
 
+        public Azure.BicepOutputReference ConnectionStringOutput { get { throw null; } }
+
         public bool IsEmulator { get { throw null; } }
+
+        [System.Diagnostics.CodeAnalysis.MemberNotNullWhen(true, "ConnectionStringSecretOutput")]
+        public bool UseAccessKeyAuthentication { get { throw null; } }
+
+        public void ApplyAzureFunctionsConfiguration(System.Collections.Generic.IDictionary<string, object> target, string connectionName) { }
```

Comment:
@captainsafia - all the rest of these implementations are explicitly implemented interface methods. But this new one isn't. Was there a reason for this? Or is that an oversight?

Reply:
Addressed the oversight in https://github.com/dotnet/aspire/pull/7667.

Reply:
https://github.com/dotnet/aspire/pull/7667